packages/jest-worker/src/__tests__/process-integration.test.ts

```diff
@@ -45,28 +42,33 @@ describe('Jest Worker Integration', () => {
   beforeEach(() => {
     mockForkedProcesses = [];
 
+    class MockChildProcess extends EventEmitter {
+      connected = true;
+      send = jest.fn();
+    }
+
     jest.mock('child_process', () => ({
       fork() {
-        const forkedProcess = mockBuildForkedProcess();
+        const forkedProcess = new MockChildProcess() as unknown as ChildProcess;
```

Comment:
why do we need to cast this? can we make the class extend (or implement) `ChildProcess`?

Reply:
Good idea. I just got rid of `ChildProcess`.